### ISSUE TITLE
Documentation: invalid instructions to execute unit tests

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6182]
|<img alt="sammyo's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| sammyo reported|
|-|-|
|Trac ID|trac#6182|
|Type|defect|
|Created|2012-11-14 13:39:46Z|
|Branch|https://github.com/twisted/twisted/tree/testing-howto-6182|

The trial/testing howto directs users to run tests from inside the package directory being tested, when it should direct them to run them from inside the directory containing that directory instead.

Attachments:

* [soo-twisted-patch.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/588/588560f81720e74c27d87a32244b2ac6a1ab977a/b5cbe0d0a6e4991b4cd6527250877c8372e01beb.patch) (666 bytes) - added by sammyo on 2012-11-14 13:41:10Z - patch to fix documentation

<details><summary>Searchable metadata</summary>

```
trac-id__6182 6182
type__defect defect
reporter__sammyo sammyo
priority__normal normal
milestone__ 
branch__branches_testing_howto_6182 branches/testing-howto-6182
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation_trial_unit_test documentation trial unit test
time__1352900386000000 1352900386000000
changetime__1352901953000000 1352901953000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [36392]) Branching to 'testing-howto-6182'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [36394]) Be consistent in the use of the idea of directory "containment"; also avoid tripping over a lore2sphinx shortcoming.

refs [#6182](https://github.com/twisted/twisted/issues/6182)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

Thanks.  I made a couple further small tweaks and will apply this now.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [36396]) Merge testing-howto-6182

Author: sammyo, exarkun
Reviewer: exarkun
Fixes: [#6182](https://github.com/twisted/twisted/issues/6182)

Correct trial usage directions to mention the right working directory to use.
